observability/micrometer/src/main/java/org/wildfly/extension/micrometer/prometheus/PrometheusContextService.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.extension.micrometer.prometheus;
+
+import static org.wildfly.extension.micrometer.prometheus.PrometheusRegistryDefinition.MICROMETER_PROMETHEUS_CONTEXT_CAPABILITY;
+
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+import org.jboss.as.controller.OperationContext;
+import org.jboss.as.server.mgmt.domain.ExtensibleHttpManagement;
+import org.jboss.msc.Service;
+import org.jboss.msc.service.ServiceBuilder;
+import org.jboss.msc.service.StartContext;
+import org.jboss.msc.service.StopContext;
+
+/**
+ * @author <a href="http://jmesnil.net/">Jeff Mesnil</a> (c) 2018 Red Hat inc.
+ */
+public class PrometheusContextService implements Service {
+    static final String HTTP_EXTENSIBILITY_CAPABILITY = "org.wildfly.management.http.extensible";
+    private final Consumer<PrometheusContextService> consumer;
+    private final Supplier<ExtensibleHttpManagement> extensibleHttpManagement;
+    private final Supplier<Boolean> securityEnabledSupplier;
+    private final String prometheusContext;
+    private final WildFlyPrometheusRegistry registry;
+
+    static void install(OperationContext operationContext,
+                        WildFlyPrometheusRegistry registry,
+                        String prometheusContext,
+                        boolean securityEnabled) {
+        ServiceBuilder<?> serviceBuilder = operationContext.getCapabilityServiceTarget()
```

Comment:
```java
operationContext.getCapabilityServiceTarget().addCapability(MICROMETER_PROMETHEUS_CONTEXT_CAPABILITY)
```

Reply:
@pferraro Am I supposed to add that line or replace line 35 with it?

Reply:
Replace